dockers/docker-fpm-frr/base_image_files/TS

```diff
@@ -4,17 +4,31 @@
 [ -f /etc/sonic/sonic-environment ] && . /etc/sonic/sonic-environment
 
 PLATFORM=${PLATFORM:-`sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`}
+switch_type=`sonic-db-cli CONFIG_DB hget 'DEVICE_METADATA|localhost' 'switch_type'`
+TSA_CHASSIS_STATE=false
+
+if [[ $switch_type == 'voq' ]]; then
```

Comment:
could you please change this to include chassis-packet or change the check to look for type == "SpineRouter"?

Reply:
@fountzou : Please address this comment.

Reply:
Thank you for comment @tjchadaga @abdosi . Fixed -- kindly see https://github.com/sonic-net/sonic-buildimage/pull/19527